packages/blade/src/components/BottomSheet/__tests__/BottomSheet.web.test.tsx

```diff
@@ -289,25 +289,27 @@ describe('<BottomSheet />', () => {
     expect(selectInput).toHaveTextContent('Select Option');
 
     // select multiple elements
+    expect(queryAllByLabelText('Close Apple tag')?.[0]).toBeFalsy();
     await user.click(getByRole('option', { name: 'Apple' }));
-    expect(selectInput).toHaveTextContent('Apple');
+    expect(queryAllByLabelText('Close Apple tag')[0]).toBeInTheDocument();
     await user.click(getByRole('option', { name: 'Orange' }));
-    expect(selectInput).toHaveTextContent('2 items selected');
-    await user.click(getByRole('option', { name: 'Banana' }));
-    expect(selectInput).toHaveTextContent('3 items selected');
+    expect(queryAllByLabelText('Close Apple tag')[0]).toBeInTheDocument();
+    expect(queryAllByLabelText('Close Orange tag')[0]).toBeInTheDocument();
 
     // close the sheet
     await user.click(getByRole('button', { name: /Close/i })!);
     expect(queryByTestId('bottomsheet-body')).not.toBeVisible();
 
-    // asssert the selected items
-    expect(selectInput).toHaveTextContent('3 items selected');
+    expect(queryAllByLabelText('Close Apple tag')[0]).toBeInTheDocument();
+    expect(queryAllByLabelText('Close Orange tag')[0]).toBeInTheDocument();
```

Comment:
TLDR of most test changes is that earlier we used to assert on input's internal text like '3 items selected', now we assert on whether the tag for that selection exists in document or not